eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -351,4 +351,13 @@ function Remove-EmptySections {
     }
     $ChangeLogEntry.ReleaseContent = $sanitizedReleaseContent.ToArray()
   }
+}
+
+function Get-TrimmedVersion
```

Comment:
While this helper is fine I was thinking about a methot that gets the latest release date or null if unreleased.